provider/info/consts.go

```diff
@@ -18,13 +18,19 @@ package info
 type ocmEnv string
 
 const (
-	ocmEnvProd  ocmEnv = "production"
-	ocmEnvStage ocmEnv = "stage"
-	ocmEnvInt   ocmEnv = "integration"
+	ocmEnvProd         ocmEnv = "production"
+	ocmEnvStage        ocmEnv = "stage"
+	ocmEnvInt          ocmEnv = "integration"
+	ocmEnvFedRAMPProd  ocmEnv = "fedramp-production"
+	ocmEnvFedRAMPStage ocmEnv = "fedramp-stage"
+	ocmEnvFedRAMPInt   ocmEnv = "fedramp-int"
 )
 
 var ocmAWSAccounts = map[ocmEnv]string{
-	ocmEnvProd:  "710019948333",
-	ocmEnvStage: "644306948063",
-	ocmEnvInt:   "896164604406",
+	ocmEnvProd:         "710019948333",
+	ocmEnvStage:        "644306948063",
+	ocmEnvInt:          "896164604406",
+	ocmEnvFedRAMPProd:  "448648337690",
```

Comment:
Is it fine to have these on GitHub? Is there another way we can store them if not?

Reply:
Does it help to know that the `rosa` folks did it first? 😄 https://github.com/openshift/rosa/blob/cca9ff58f9eeffdd5de120defd61e0fe7d096fd0/pkg/fedramp/config.go#L37

Reply:
yes it does 😛